Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -2792,12 +3169,110 @@ def get_remote_data_command(client: Client, params: Dict[str, Any], args: Dict)
             'ContentsFormat': EntryFormat.JSON
         })
 
+    failure_message = 'Failed communicating with long running container.'
+    if mirror_options == MIRROR_OFFENSE_AND_EVENTS:
+        offenses_waiting_for_update = context_data.get(MIRRORED_OFFENSES_CTX_KEY, [])
+        max_retries = MAX_FETCH_EVENT_RETIRES * (len(offenses_waiting_for_update) + 3)
+        offense_to_remove = None
+        is_waiting_to_be_updated = True
+        evented_offense = None
+        retries = 0
+        while ((not evented_offense) or is_waiting_to_be_updated) and retries < max_retries:
+            if retries != 0:
+                time.sleep(FAILURE_SLEEP)
```

Comment:
I am a bit afraid we might receive timeout here because the remote data docker container is 5m i think? WDYT?